src/helpers/formatVerificationFieldString.ts

```diff
@@ -0,0 +1,14 @@
+export const formatVerificationFieldString = (input: string) => {
```

Comment:
It looks like `input` can be an empty string/optional (`{formatVerificationFieldString(details.verificationField ?? "")}`). Should we reflect that here? That way, we could optimize by returning the same input early if it is an empty string.

Reply:
![image](https://github.com/stellar/stellar-disbursement-platform-frontend/assets/7346473/c21c30f9-dc51-475b-aa0a-3f1ae18a81e0)

The programmatic formatting is not great as we're not making uppercase words that should be (for example, PIN and ID). Maybe we should do manual mapping to labels instead? It would also be better for translations if they are added later. :thinking: What do you think?

Something like:
```json
{
  "DATE_OF_BIRTH": "Date of Birth",
  "PIN": "PIN",
  "NATIONAL_ID_NUMBER": "National ID Number"
}
```

Reply:
Great idea! Thanks @quietbits